elections/tac-and-scir-election-process.md

```diff
@@ -0,0 +1,66 @@
+Eligible community members, according to the same criteria defined above for voter eligibility, can self-nominate using the form below.
+
+[TAC Candidate Self-Nomination Form](https://docs.google.com/forms/d/e/1FAIpQLSdxUbbfJGgjU_cZ_uGvUm-5ePoXu7q2pGiNyH2G8euwpn23Sw/viewform)
+
+
+## SCIR Self-Nomination Information and Process
+
+Since early in its existence, the OpenSSF Governing Board has sought to ensure it gets adequate input from voices in the software security community who would otherwise not be at the table.
+We seek candidates for the Security Community Individual Representative (SCIR) who can represent those voices, while also being a subject matter expert in the field with their own set of perspectives.
+Familiarity with the different OpenSSF working groups and projects, and being able to dedicate the time to be sufficiently informed on the issues that arise in our monthly calls and ongoing discussions, is highly desired.
+It is also highly desired, but not required, that the SCIR be an eligible OpenSSF community member by the criteria defined above.
+
+[SCIR Candidate Self-Nomination Form](https://docs.google.com/forms/d/e/1FAIpQLScoabVVjW1kvTNcKXhTEdvxQd13Caql-I36MOixHVO1dLwIiQ/viewform)
+
+
+## Election Process
+
+The Election Process will be monitored by Election Officials, who
+* review nominees and confirm their eligibility,
+* approve the election timeline, and
+* review the ballot request form and validate voter eligibility.
+
+Election Officials are eligible community members who have volunteered to oversee and help ensure the integrity of the election process. By volunteering in this role, they recuse themselves from running for a TAC or SCIR position but they still are allowed to vote in the election.
+
+Candidates for each election will be reviewed by the Election Officials to ensure their eligibility per the guidelines above. At the end of the validation period, the complete list of candidates will be published in this repo.
+
+Similarly, the pool of eligible voters will be reviewed by Election Officials to ensure that all voters meet the eligibility criteria established above If a ballot requester is considered ineligible, an Election Official will follow up directly with the individual.
+
+If there are more nominees than positions for either the TAC or the SCIR, the election will be conducted using an electronic voting tool.
+The [Single Transferable Vote](https://en.wikipedia.org/wiki/Single_transferable_vote) method will be used to vote for TAC members as there are multiple candidates for multiple positions.
+Eligible voters will receive an invitation to access their ballot from Opavote.
+The ballot will include the option to vote for both the TAC and SCIR.
+
+Questions related to the nomination and/or election process should be directed to [operations@openssf.org](mailto:operations@openssf.org).
+
+
+## 2023 Election Timeline
+
```

Comment:
```suggestion

The timeline in a standard election will follow the below format:

10 months after the previous election, an election committee will be established by ...(a member of staff? a call for volunteers to $mailinglist?). The volunteers have two weeks to begin the election process.

- Voter Ballot Request Period, and TAC and SCIR Self-Nomination Period - 2 weeks (concurrently)
- Validation Period - 1 Week
- Voting Period - 2 Weeks
- Announcement of results - Day after the conclusion of the Voting Period

```

Reply:
Sounds good to me - including the proposals in parenthesis.